README.md

```diff
@@ -64,3 +65,27 @@ puppet-task run mco_rpc $TARGETS agent=package action=install arguments='package
 
 This is a pre `1.0.0` release and future versions may have breaking changes.
 This relies on the Puppet agent packaged ruby and the MCO gem to run.
+
+## Known Issues
+Puppet Enterprise 2018.1 is the last release to support MCollective. While PE 2018.1 remains supported, Puppet will continue to address security issues for MCollective. Feature development has been discontinued. Future releases of PE will not include MCollective. For more information, see the [Puppet Enterprise support lifecycle](https://puppet.com/misc/puppet-enterprise-lifecycle).
+
+If you use `mco` commands to call agents, set up the puppetlabs/mco_rpc module and create individual Puppet tasks to replace each command that you use.
+
+### Run MCollective RPC agents through Puppet tasks
+
+1. To use the puppetlabs/mco_rpc module, add this declaration to your Puppetfile:
+    ```
+    mod 'puppetlabs-mco_rpc', '0.1.0'
+    ```
+2. In the PE console, run a task instead of using the mco command. For example, if you use the command `mco puppet disable` replace it with the following Puppet task:
+  
+  a. In the Run section, click **Task**.
```

Comment:
These aren't formatting very well in markdown. Look at the source for the following example

1. First entry
   1. First sub-entry
   1. Second sub-entry
1. Second entry